requirements.txt

```diff
@@ -1,5 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
-#rpm-python
+# This can not be installed from pip, on Fedora you need to install python3-rpm.
+# https://github.com/rpm-software-management/rpm/tree/master/python/rpm
+#python3-rpm
```

Comment:
@nforro 

> So I think it would be better to wait until F24 EOL before doing this change.

Now F24 is end of life.
https://fedoraproject.org/wiki/End_of_life